tests/test_functons.py

```diff
@@ -0,0 +1,48 @@
+import sys
+
+import pytest
+
+from codecademy.functions import one_good_turn
+from codecademy.functions import square
+from codecademy.functions import deserves_another
+from codecademy.functions import biggest_number
```

Comment:
please sort imports alphabetically - refer to my original comment for details and link to PEP8